src/views/instances/new-dhis2/fields/boolean-parameter-checkbox.tsx

```diff
@@ -1,9 +1,21 @@
 import { CheckboxFieldFF } from '@dhis2/ui'
 import { Field } from 'react-final-form'
-import { formatBool, parseBool } from './helpers'
 import styles from './fields.module.css'
 import { ParameterFieldProps } from './parameter-field'
+import { FC } from 'react'
 
-export const BooleanParameterCheckbox = ({ displayName, parameterName }: ParameterFieldProps) => (
-    <Field type="checkbox" format={formatBool} parse={parseBool} required name={parameterName} label={displayName} component={CheckboxFieldFF} className={styles.parameterCheckbox} />
+export const parseBool = (bool: any): string => (typeof bool === 'boolean' && bool ? 'true' : 'false')
+export const formatBool = (str: string): boolean => str === 'true'
+
+export const BooleanParameterCheckbox: FC<ParameterFieldProps> = ({ displayName, parameterName, stackId }) => (
```

Comment:
what version of react is in use? It's just that this way of typing react components has changed, from what I remember, back in react 16...